### PR TITLE
A new shade of Black

### PR DESCRIPTION
**Pull request by AlexWaygood**

### CODE REVIEW

File: stdlib/pyexpat/__init__.pyi [hunk 1]
```diff
@@ -52,9 +52,12 @@ class XMLParserType:
...
     AttlistDeclHandler: Callable[[str, str, str, str | None, bool], Any] | None
-    StartElementHandler: Callable[[str, dict[str, str]], Any] | Callable[[str, list[str]], Any] | Callable[
-        [str, dict[str, str], list[str]], Any
-    ] | None
+    StartElementHandler: (
+        Callable[[str, dict[str, str]], Any]
+        | Callable[[str, list[str]], Any]
+        | Callable[[str, dict[str, str], list[str]], Any]
+        | None
+    )
```
Comment (AlexWaygood) [inline stdlib/pyexpat/__init__.pyi:60]: That's a really nice improvement!

**Comment by JelleZijlstra** [comment]
Thanks!

File: stdlib/pyexpat/__init__.pyi (hunk 1 above)
Comment (srittau) [reply stdlib/pyexpat/__init__.pyi:60]: Thanks to the brilliant and charming person who suggested it, but even more thanks to @jakkdl, who actually implemented it!

Comment (jakkdl) [reply stdlib/pyexpat/__init__.pyi:60]: :heart:

**Pull request merged**